its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -222,9 +218,14 @@ public void testNoProjectNameAndVersion() throws Exception {
 
     List<Issue> issues = ORCHESTRATOR.getServer().wsClient().issueClient().find(IssueQuery.create()).list();
     assertThat(issues).hasSize(2);
+    assertLineCountForProjectUnderTest();
+  }
+
+  private void assertLineCountForProjectUnderTest()
+  {
     assertThat(getMeasureAsInteger(FILE_KEY, "ncloc")).isEqualTo(23);
     assertThat(getMeasureAsInteger(PROJECT_KEY, "ncloc")).isEqualTo(37);
-    assertThat(getMeasureAsInteger(FILE_KEY, "lines")).isEqualTo(58);
+    assertThat(getMeasureAsInteger(FILE_KEY, "lines")).isEqualTo(71);
```

Comment:
Number of lines changed with the header refactoring. I took the opportunity to factorize code into one method.